Vagrantfile

```diff
@@ -12,20 +12,63 @@ end
 # Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
 VAGRANTFILE_API_VERSION = "2"
 
+def get_amis
+  require "open-uri"
+  require "json"
+
+  images_uri = "https://dl.flynn.io/ec2/images.json"
+  json = JSON.parse(open(images_uri).read())
+  latest = json["versions"][0]["images"]
+
+  Hash[ latest.map {|ami| [ ami["region"], ami["id"] ] } ]
```

Comment:
This assumes that there will only be one AMI per region. When we add instance-store AMIs (#977, this will not be the case). I think we need to divide it up by disk type as well.